src/main/java/teamrtg/rtg/core/RTG.java

```diff
@@ -38,11 +44,12 @@ public void preInit(FMLPreInitializationEvent event) {
 
         configPath = event.getModConfigurationDirectory() + "/RTG/";
         Mods.syncAllConfigs();
+        
+        // This must come before the event manager is registered.
+        this.registerStructures();
 
+        Logger.info("[FMLPreInitializationEvent] Creating RTG's EventManager");
         eventMgr = new EventManagerRTG();
-        MinecraftForge.EVENT_BUS.register(eventMgr);
-        MinecraftForge.ORE_GEN_BUS.register(eventMgr);
-        MinecraftForge.TERRAIN_GEN_BUS.register(eventMgr);
 
         worldtype = new WorldTypeRTG("RTG");
```

Comment:
Meh.. I need to redo this again.. I swear I've fixed this like 10 times now and everytime I end up not getting the commit pushed.

`worldtype = new WorldTypeRTG(ModInfo.MOD_ID)`